src/services/api.ts

```diff
@@ -61,47 +59,13 @@ export enum Errors {
 	InvalidTokenId = "Invalid token id",
 }
 export default class Contract {
-	api: ApiInterface;
-	tokenCache: Map<TokenId, Token> = new Map<TokenId, Token>();
-	tokenList: TokenId[] = new Array<TokenId>();
-	lastUpdated = 0;
-
-	constructor() {
-		let mocked = CONTRACT_MOCKED;
-		if (typeof window !== "undefined" && window.localStorage) {
```

Comment:
Bohdan, I have fixed the tests by adding an argument `isMocked` to the page page level component. But that prevents the build process because the page level component should not have an argument... if you delete those code for deciding either using GgxNetwork(api!) for real browsers or using GgxNetworkMock() for testing, how can I fix this? I think we should add those code back somehow. Okay?